src/functionalTest/java/uk/gov/hmcts/darts/usermanagement/UserManagementFunctionalTest.java

```diff
@@ -80,9 +84,54 @@ void shouldModifyUser() {
         );
     }
 
+    @Test
+    void getUserByEmail() {
+        createUser();
+
+        Response response = buildRequestWithExternalGlobalAccessAuth()
+            .baseUri(getUri(ADMIN_USERS))
+            .queryParam(COURTHOUSE_ID, 21)
+            .header(EMAIL_ADDRESS, "james.smith@hmcts.net")
+            .get()
+            .thenReturn();
+
+        assertEquals(200, response.getStatusCode());
```

Comment:
Please can you also check the response body to ensure that it's returning the expected user?

Reply:
assert true for response email added